analytics/location_heatmaps/geo_utils.py

```diff
@@ -278,141 +296,124 @@ def split_regions(tree_prefix_list,
   Args:
       tree_prefix_list: matches vector id to the tree prefix.
       vector_counts: vector values aggregated from the users.
-      threshold: threshold value used to split the nodes.
+      split_threshold: threshold value used to split the nodes.
       image_bit_level: stopping criteria once the final resolution is reached.
       collapse_threshold: threshold value used to collapse the nodes.
+      expand_all: expand all regions,
+      last_result: use previous level results to compute conf intervals,
+      count_min: use count-min sketch
   Returns:
-      new_tree, new_tree_prefix_list, finished
+      new_tree, new_tree_prefix_list, fresh_expand
```

Comment:
The meaning of fresh_expand is not obvious, especially since this function both collapses and expands. Maybe num_newly_expanded_nodes?

Reply:
great, thanks a lot!